packages/zoe/src/contractSupport/percentMath.js

```diff
@@ -5,11 +5,17 @@ import { natSafeMath } from './index';
 const { multiply, floorDivide } = natSafeMath;
 
 // make a percentMath object, which represents a unitless fraction between 0
-// and 1. It can be multiplied by ERTP amounts to take a percentage. inverse()
-// produces a new percent object that is the  original's additive complement.
+// and 1. It can be multiplied by ERTP amounts to take a percentage.
+// complement() produces a new percent object that is the complement of the
+// original.
```

Comment:
You're using the word "complement" in the explanation, which is not clear enough by itself. Example? Like "The complement of 20% is 80% rather than 500%"?